### DISCUSSION TITLE
Change hostname from settings

### DISCUSSION THREAD

**Discussion by Canonius** [jomjol/AI-on-the-edge-device#2656] [General] [answered]
### The Feature

I want to be able to change the hostname without to disassemble the setup.

**Comment by Slider0007** [comment, marked as answer]
This is already implemented. Activate Expert View and adapt hostname from WebUI.

Please have a look to documentation: https://jomjol.github.io/AI-on-the-edge-device-docs/Parameters/#parameter-hostname

If this is what you're requesting, please close the request.

**Comment by Canonius** [comment]
Exactly what I was looking for! 👍 thanks

**Comment by razzeee** [comment]
nice, did not find it, cause I was using the browser search to look for `watermeter`, which seems to be the fallback
